go.mod

```diff
@@ -38,6 +38,8 @@ require (
 	github.com/jaguilar/vt100 v0.0.0-20150826170717-2703a27b14ea
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/moby/locker v1.0.1
+	github.com/moby/sys/mount v0.1.1 // indirect
+	github.com/moby/sys/mountinfo v0.4.0 // indirect
```

Comment:
I wasn't sure if this is the "correct" way to do this. Basically, `go mod` by itself picked older versions of these dependencies (I guess because the dependencies using them are not using go modules). However, quite some perfomance improvements went into `mountinfo`, as well as some breaking changes in `mount` (renaming of some fields).

With this, I'm trying to achieve that we get the improved versions, as well as verifying that the breaking changes would not break how it's used in this codebase (and thus, re-vendoring buildkit in docker/docker)

Reply:
Did gomod add the `indirect` comment?

Reply:
No, go mod did not add these to the list, and selected old versions.

I haven't checked what happens if I remove the `// indirect` comment (or replace it with a different comment). Let me give that a quick try

Reply:
If I change the comment to;

```go
// (indirect) force more current version of sys/mountinfo than go mod selects
```

go mod updates it to

```go
// indirect; (indirect) force more current version of sys/mountinfo than go mod selects
```

So it recognises the `// indirect` prefix, and any _custom_ comment after it should be separated with a semicolon (`;`)

I updated the comments to the above